### PR TITLE
refactor: update types according to AIP36 changes

### PR DESCRIPTION
**Pull request by brenopolanski**
## Summary

- [x] Update types according to AIP36 changes - https://github.com/ArkEcosystem/core/blob/2.7/packages/core-magistrate-crypto/src/enums.ts
- [x] Check undefined transaction types

## Checklist

- [ ] Documentation _(if necessary)_
- [ ] Tests _(if necessary)_
- [x] Ready to be merged

### CODE REVIEW

File: __tests__/unit/specs/mixins/transaction-types.spec.ts [hunk 1]
```diff
@@ -48,91 +48,91 @@ describe("Mixins > Transaction Types", () => {
 
       expect(wrapper.vm.isBusinessEntityRegistration(MagistrateTransaction.ENTITY, TypeGroupTransaction.MAGISTRATE, {
         type: MagistrateTransactionEntityType.BUSINESS,
-        subType: MagistrateTransactionEntitySubType.NONE,
+        // subType: MagistrateTransactionEntitySubType.NONE,
```
Comment (ItsANameToo) [inline __tests__/unit/specs/mixins/transaction-types.spec.ts]: why are these commented? remove them if they are no longer needed

Comment (brenopolanski) [reply __tests__/unit/specs/mixins/transaction-types.spec.ts]: There will be new subTypes and so the comments. But if you prefer, i can remove it for now

Comment (ItsANameToo) [reply __tests__/unit/specs/mixins/transaction-types.spec.ts]: remove them

Comment (brenopolanski) [reply __tests__/unit/specs/mixins/transaction-types.spec.ts]: okay :+1:

**Review by ItsANameToo — APPROVED** [review]

**Pull request merged**